secant/Wrappers/WrappedSynchronizer.swift

```diff
@@ -8,26 +8,51 @@
 import Foundation
 import ZcashLightClientKit
 import Combine
+import ComposableArchitecture
+
+enum WrappedSynchronizerState: Equatable {
+    case unknown
+    
+    case transactionsUpdated
+    case started
+    case progressUpdated
+    case statusWillUpdate
+    case synced
+    case stopped
+    case disconnected
+    case syncing
+    case downloading
+    case validating
+    case scanning
+    case enhancing
+    case fetching
+    case minedTransaction
+    case foundTransactions
```

Comment:
What would be the difference between mined transaction and foundTransactions?

We are currently not using the mempool to fetch unmined transactions so every transaction not originated on this wallet will be detected when mined.

Reply:
as agreed on slack, we need to support both